tests/test_filesystem.py

```diff
@@ -64,6 +64,7 @@ class TestFilesystem(object):
         ),
         'truncate -s 0 /tmp/file_to_flush': (0, '', ''),
         'mv /tmp/source /tmp/destination': (0, '', ''),
+        'cp /tmp/source /tmp/destination': (0, '', '')
```

Comment:
on line https://github.com/rhevm-qe-automation/python-rrmngmnt/pull/92/files#diff-c3f98a9eed559eadf88da12842f84025R148

you are using isdir, which also executes command, '[ -d /tmp/source ]' so you need to add it as well ...